ix-dev/community/node-red/app.yaml

```diff
@@ -0,0 +1,32 @@
+app_version: 3.1.10
```

Comment:
Just curious how the app version is being deduced here

Reply:
If I recall correctly the "default" image was with node 16, and we used that as a "base" to get the app_version.